encoding/json/sonic_benchmark_test.go

```diff
@@ -0,0 +1,13 @@
+//go:build sonic && !(darwin || arm64)
+
+package json
+
+import "testing"
+
+// BenchmarkUnmarshal-16  1900803  631.8 ns/op
+// Usage: go test --tags=sonic -bench=BenchmarkUnmarshal -v
+func BenchmarkUnmarshal(b *testing.B) {
```

Comment:
Only one of these is needed since they're the same test and doesn't matter what the build tag is
You can keep both to be explicit if you want, but if the build tag changes the json processor changes regardless

Reply:
[addressed](https://github.com/thrasher-corp/gocryptotrader/pull/1623/commits/3905b625d959af4a2784355f90f72630a4d041ed)